apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector/preview.scss

```diff
@@ -0,0 +1,189 @@
+@import '~@wordpress/base-styles/variables';
```

Comment:
Since https://github.com/WordPress/gutenberg/pull/19159 is an issue at the moment, you could also import the whole set via this file:

https://github.com/Automattic/wp-calypso/blob/f2b1724708c10bc1980be2228f597340cc9b64b2/client/assets/stylesheets/gutenberg-base-styles.scss#L1-L6

Reply:
I pushed be80333 which seems to be enough.